x/wasm/keeper/msg_dispatcher.go

```diff
@@ -85,9 +85,13 @@ func (d MessageDispatcher) DispatchSubmessages(ctx sdk.Context, contractAddr sdk
 
 		// Prevent the use of fswap and fbridge module in Submessages
 		// https://github.com/Finschia/finschia-sdk/pull/1336, https://github.com/Finschia/finschia-sdk/pull/1340
-		if stargateMsg := msg.Msg.Stargate; stargateMsg != nil &&
-			(strings.Contains(stargateMsg.TypeURL, "lbm.fswap.v1") || strings.Contains(stargateMsg.TypeURL, "lbm.fbridge.v1")) {
-			return nil, sdkerrors.Wrap(types.ErrUnsupportedForContract, "fswap and fbridge not supported of Stargate")
+		if stargateMsg := msg.Msg.Stargate; stargateMsg != nil {
+			if strings.HasPrefix(stargateMsg.TypeURL, "/lbm.fswap.v1") {
+				return nil, sdkerrors.Wrap(types.ErrUnsupportedForContract, "fswap not supported of Stargate")
+			}
+			if strings.HasPrefix(stargateMsg.TypeURL, "/lbm.fbridge.v1") {
+				return nil, sdkerrors.Wrap(types.ErrUnsupportedForContract, "fbridge not supported of Stargate")
```

Comment:
supported "by" or "with" looks better.

Reply:
https://github.com/Finschia/wasmd/pull/123/commits/58714d1fb3f9f33fab3f3e69f19bd4d88320f1cf Updated, thank you.